src/App.css

```diff
@@ -0,0 +1,63 @@
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.1/font/bootstrap-icons.css");
```

Comment:
- [ ] OPTIONAL: While your styling and your project project are very functional, it does not meet the styling requirements.  I'd advise you to meet this particular requirement before the final evaluation.

If something happened during development (like a teammate leaving) kindly add an explanation to your PR description according to this:

![image](https://github.com/PabloEGonz/full-stack-capstone-front-end/assets/57421823/01fdb78c-9d71-4082-98b7-26acc933c247)